main.tex

```diff
@@ -120,8 +120,8 @@ \section{Introduction} \label{sec:intro}
 packages are available, which can help with tasks that are not astronomy specific but
 might be performed in the course of astronomical research; e.g., interfacing with
 databases or statistical inferences. More recently, the development of package managers such as
-Anaconda\footnote{\url{https://anaconda.org/}} have streamlined the installation process for
-most packages, significantly reducing the entry barrier for using many such libraries.
+Anaconda\footnote{\url{https://anaconda.org/}} has significantly streamlined the installation
+process for many libraries, lowering any barriers to entry.
```

Comment:
I think "lowering any barriers to entry" is too strong a statement. Maybe just change to "lowering many barriers to entry"?

Reply:
Done!